sdformat_to_mjcf/sdformat_to_mjcf/sdformat2mjcf.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+import sdformat as sdf
+from dm_control import mjcf
+from sdformat_mjcf.converters.model import add_model
+from sdformat_mjcf.converters.world import add_world
+
+
+def sdformat_root_to_mjcf(root):
+    mjcf_root = mjcf.RootElement()
+    mjcf_root.compiler.eulerseq = 'XYZ'
+    mjcf_root.model = root.model().name()
+    asset = mjcf_root.asset
+    asset.add("texture",
+              name="grid",
+              type="2d",
+              builtin="checker",
+              width=512,
+              height=512,
+              rgb1=[.1, .2, .3],
+              rgb2=[.2, .3, .4])
+    asset.add("material",
+              name="grid",
+              texture="grid",
+              texrepeat="1 1",
+              texuniform=True,
+              reflectance=0.2)
+
+    mjcf_root.worldbody.add("geom",
+                            name="floor",
+                            size=[0, 0, .05],
+                            pos=[0, 0, -1],
+                            type="plane",
+                            material="grid",
+                            condim=3)
+
+    if root.model():
+        return add_model(mjcf_root, root.model())
+    elif root.world_count() > 0:
+        # Does not support multiple worlds
+        return add_world(mjcf_root, root.world_by_index(0))
+
+
+def sdformat_file_to_mjcf(model_file):
+    root = sdf.Root()
+    errors = root.load(model_file)
+    if errors:
+        print(errors)
+    else:
+        mjcf_root = sdformat_root_to_mjcf(root)
+        print(mjcf_root.to_xml_string())
```

Comment:
```suggestion
        export_with_assets(mjcf_root, ".", output_file)
```

you should import `from dm_control.mjcf.export_with_assets import export_with_assets`

Reply:
Done in https://github.com/gazebosim/gz-mujoco/pull/6/commits/a151a0c6cb0ca95a1a0f2c070205c192a4f738a9. Instead of the current directory, I used the directory of the output file. What do you think?